pom.xml

```diff
@@ -224,6 +230,12 @@ SOFTWARE.
       <scope>compile</scope>
       <optional>true</optional>
     </dependency>
+    <dependency>
```

Comment:
@victornoel do we really need this dependency for tests?

Reply:
@g4s8 I think it makes sense to be able to see what are the logs when running tests. I always add it to my projects for tests. But I have no problem to remove it if you want?

Reply:
@victornoel right, let's keep it